sig-vmware/charter.md

```diff
@@ -0,0 +1,58 @@
+- Desktop hypervisors (Fusion for OS X and Workstation for Linux and Windows), as supported by Minikube.
+- vCloud Director, which enables service providers to provision and manage multi tenant clouds, which can host Kubernetes.
+
+
+### In scope
+- Deploying Kubernetes of any type (standard or a packaged distribution from any vendor) on vSphere infrastructure.
+- Utilizing networking and storage features of VMware infrastructure through Kubernetes.
+- Determining and documenting best practices for configuring Kubernetes when running on vSphere infrastructure.
+- Determining and documenting best practices for configuring VMware infrastructure when supporting Kubernetes.
+- Determining and documenting best practices for using common configuration management and orchestration tools for deploying and managing Kubernetes on VMware infrastructure.
+- Discussing bugs and feature requests recorded as Kubernetes or VMware cloud provider issues on GitHub. These issues should be tagged with ``sig/vmware``.
+
+A directory of specialized subject area GitHub teams for issues, PRs, and design proposals is defined in [SIG GitHub Teams](https://github.com/kubernetes/community/tree/master/sig-vmware#github-teams).
+
+#### Code, Binaries and Services
+
+- vSphere Cloud Provider
+- vRealize Automation Cluster API Provider
+
+Code locations and projects are defined in [SIG Subprojects](https://github.com/kubernetes/community/tree/master/sig-vmware#subprojects)
+
+#### Cross-cutting and Externally Facing Processes
+
+SIG VMware serves to bring together members of the VMware and Kubernetes community to maintain, support and provide guidance for running Kubernetes on VMware platforms.
+
+The VMware SIG will provide a forum for hosting related architectural planning and discussion. Associated activities related to development, testing, and documentation will be tracked and reported by the SIG.
+
+- The vSphere platform’s availability and resource management can create pools that extend across Kubernetes worker node, management node, and namespace boundaries. Unique features of the vSphere platform related to high availability, load balancing, and resource management (storage, networking, compute) can supplement existing capabilities of Kubernetes, container runtime, and OS functionality.
+  - Users can do some of this via configuration today, but opportunities exist to achieve more efficient, predictable, and dependable service through additional new CRDs, plug-ins, and enhancement proposals (KEPs) related to other Kubernetes components.
+  - Interaction is expected to cut across multiple aspects of Kubernetes in ways not firmly bounded by existing SIG dividing lines (e.g. node, scheduling, scalability).
+

```

Comment:
The first paragraph looks good, as an external process description.  The rest of this section reads to me more like marketing for vSphere than it does a description of how this SIG interacts with other SIGs on cross-cutting efforts.

Seems like this is all justification for why the SIG should continue to exist outside the scope of other SIGs, which IMO could be articulated in scope above, eg:

- "Hosting architectural planning and discussion related to new CRDs, plug-ins and KEPs that allow the vSphere platform to supplement the existing capabilities of Kubernetes, container runtime, and OS functionality"

In here, I would expect to see something more like:
- [We co-own vSphere-specific code related to cluster and machine provisioning with SIG Cluster Lifecycle](https://github.com/kubernetes/community/blob/master/sig-cluster-lifecycle/charter.md#cross-cutting-and-externally-facing-processes)
- We drive changes that cross existing SIG boundaries by collaborating with the appropriate SIGs (eg: node, schedule, scalability, etc.)

Reply:
edited charter in response to your comments, adding more details on interaction with sig-cluster-lifecycle, sig-cloud-provider, and sig-storage and removing "marketing text"